symfony/plugins/orangehrmTimePlugin/dao/CustomerDao.php

```diff
@@ -17,51 +17,62 @@
  * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
  * Boston, MA  02110-1301, USA
  */
-class CustomerDao extends BaseDao {
 
-    /**
-     *
-     * @param type $limit
-     * @param type $offset
-     * @param type $sortField
-     * @param type $sortOrder
-     * @param type $activeOnly
-     * @return type 
-     */
-    public function getCustomerList($limit = 50, $offset = 0, $sortField = 'name', $sortOrder = 'ASC', $activeOnly = true) {
+namespace OrangeHRM\Time\Dao;
```

Comment:
hope you set `git config --global core.filemode false`,  `git config core.filemode false` within the repository

Reply:
done